### ISSUE TITLE
False positive for 'final-newline' rule on windows

### ISSUE THREAD

**Issue by davidgilbertson** [sasstools/sass-lint#65]
The `final-newline` rule is always reporting a warning for me and I believe it's because of the way Windows reports newlines.

Because they are actually `\r\n`, [this line](https://github.com/sasstools/sass-lint/blob/master/lib/rules/final-newline.js#L27) (`last.content !== '\n'`) is always true.

If I test for `\r\n`as well the rule behaves as expected on Windows (although I don't have a mac handy to test it on).
```js
if (last.content !== '\n' && last.content !== '\r\n' && parser.options.include) {
```
Or a shorter version...
```js
if (!last.content.match(/\r?\n/) && parser.options.include) {
```

**Comment by davidgilbertson** [comment]
It looks like the issue for `empty-line-between-blocks`. [This line](https://github.com/sasstools/sass-lint/blob/master/lib/rules/empty-line-between-blocks.js#L26) might need to be this to play well with Windows:
```js
space = (previous.content.indexOf('\n\n') >= 0 || previous.content.indexOf('\r\n\r\n') >= 0);
```

**Comment by davidgilbertson** [comment]
Or... having just learnt that `os.EOL` [is a thing](https://nodejs.org/api/os.html#os_os_eol)
```js
var os = require('os');
if (last.content !== os.EOL && parser.options.include) {
    // etc.
}
```
Nice and descriptive...
